internal/cli/events/project/list_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build unit
+
+package project
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/mongodb/mongocli/internal/mocks"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+func TestList_Run(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mockStore := mocks.NewMockEventLister(ctrl)
+	defer ctrl.Finish()
+
+	expected := &mongodbatlas.EventResponse{}
+
+	listOpts := &ListOpts{
+		store: mockStore,
+	}
+
+	listOpts.projectID = "1"
+	mockStore.
+		EXPECT().ProjectEvents(listOpts.projectID, listOpts.newEventListOptions()).
+		Return(expected, nil).
+		Times(1)
+
+	if err := listOpts.Run(); err != nil {
+		t.Fatalf("Run() unexpected error: %v", err)
+	}
+}
```

Comment:
we should check the flags used by this command with a `TestListBuilder`. See for an example https://github.com/mongodb/mongocli/blob/6ac1f831b2a7d4ae60f4a7d025e4ab3c8da50d79/internal/cli/atlas/clusters/list_test.go#L51